SwiftWisdomTests/StandardLibrary/String/String+IndexingTests.swift

```diff
@@ -2,6 +2,30 @@ import XCTest
 
 class String_IndexingTests: XCTestCase {
     
+    func testRangeSubscriptReturnsProperString() {
+        XCTAssertEqual("hello"[0..<0], "")
```

Comment:
Looks like it's just using `characters` property on string, no special indexing by bytes, so it's probably fine as is.